README.md

```diff
@@ -120,6 +121,34 @@ il_trafficgen stops automatically, after test-duration.
 If test-duration is set to 0, packet transmission happens indefinitely. So, use 'stp' command to stop packet tranmission
 Logs for the current run will be stored in /opt/il_trafficgen/pktgen/autotest/log/il_trafficgen-data-plane_debug_<DD_MM_YYYY_HH_MM_S>.log
 
+CLI MODE
+Run 'il_nperf_cli.sh' located in /opt/il_trafficgen/pktgen
+
+	1. Start il_trafficgen (as generator)in one terminal
+	cd /opt/il_trafficgen/pktgen
+	./il_nperf_cli.sh –g
+
+	wait for the il_trafficgen command prompt to appear on screen.
+
+	2. Start il_trafficgen (as responder)in another terminal
+	cd /opt/il_trafficgen/pktgen
+	./il_nperf_cli.sh –r
+
+	Wait for the il_trafficgen command prompt to appear on screen.
+
+	3. To send uplink traffic
+	Send the following command to generate Uplink traffic(i.e., P0-Tx --> P1-Rx) on GENERATOR.
+	$ socat - TCP4:localhost:2222 < start_il_trafficgen.lua
+
+	4. To send downlink traffic
+	Send the following command to generate Downlink traffic(i.e., P0-Rx <-- P1-Tx) on RESPONDER.
+	$ socat - TCP4:localhost:3333 < start_il_trafficgen.lua
+
+	5. To send traffic uplink and downlink run step 3 and 4 simultaneously.
+
+	6. Send the following command to quit both GENERATOR and RESPONDER.
+	$ socat - TCP4:localhost:2222 < quit_il_trafficgen.lua ; socat - TCP4:localhost:3333 < quit_il_trafficgen.lua
```

Comment:
"socat - TCP4:localhost:2222 < quit_il_trafficgen.lua" - would be nice to get the traffic statistics returned on port 2222 after stopping the generator. This way we don't have look the pktgen screen to see the results.

Reply:
Traffic statistics are always logged to output files in `log/` subdirectory. We don't really need to look at the pktgen screen. In fact, if the cli mode is used to run il_trafficgen, the user  won't need to see the screen at all.